src/components/cards/Video/styles.tsx

```diff
@@ -1,12 +1,14 @@
 import { StyleSheet } from 'react-native';
 import { TRANSPARENT_DARK_GREY } from '../../../styles/colors';
 import { MARGIN, CARD_MEDIA_MAX_HEIGHT, BORDER_RADIUS, ICON } from '../../../styles/metrics';
+import { isWeb } from '../../../core/data/constants';
 
 const styles = (isMediaLoading: boolean) => StyleSheet.create({
   media: {
     height: CARD_MEDIA_MAX_HEIGHT,
     marginBottom: MARGIN.LG,
     display: !isMediaLoading ? 'flex' : 'none',
+    position: isWeb ? 'relative' : undefined,
```

Comment:
Que penses-tu de plutot l'ecrire comme suit : ...isWeb && { position: 'relative' }, pour ajouter position uniquement quand il faut ?

Reply:
ça pète sur app mobile de faire ça mais j'ai l'impression qu'on peut laisser relative tout le temps en revanche

Reply:
Ah oui bien vu. En fait je me fiais a "set position to relative, but only on Web, because if you do this for Android/iOS, it will break the video and it won't display." mais en effet ca fonctionne aussi sur mon tel